packages/components/tag/src/tag.stories.tsx

```diff
@@ -10,19 +10,46 @@ export default meta;
 
 type Story = StoryObj<typeof Tag>;
 
-/** Displays a single `<Tag/>`.*/
-export const BasicExample: Story = (args: TTagProps) => {
-  return (
-    <Router>
-      <Tag {...args} />
-    </Router>
-  );
+/** Display a plain tag, unlinked and without any actions attached */
+export const BasicExample: Story = {
+  render: (args) => {
+    return (
+      <Router>
+        <Tag {...args} />
+      </Router>
+    );
+  },
+  args: {
+    tone: 'primary',
+    children: 'Ice Cream',
+    onRemove: undefined,
+    onClick: undefined,
+  },
 };
 
-BasicExample.args = {
-  type: 'normal',
-  isDisabled: false,
-  to: '/project-key/products/icecream',
-  onRemove: () => alert('Remove tag request'),
-  children: 'Ice Cream',
+/** displays the tag as a react-router link, (no hover effects) */
+export const LinkedTag: Story = {
```

Comment:
I added more stories, primarily to make sure that the existing behavior stays the same (e.g. unlike tags with `onClick` handler, links don't have a hover-effect).